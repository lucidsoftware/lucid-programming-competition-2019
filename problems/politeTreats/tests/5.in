creepy poisonous-looking ghost caramel apple 69635
frightening quadruple rasberry mummy caramel apple 73734
chilling crushed goblin gummy 77831
horrendous orange-flavored hocus pocus pumpkin 92172
horrendous foul-looking poltergeist cottoncandy 4112
unnerving double chocolate pumpkin pure sucrose 36886
spooky magical vampire creme de la crumb 8214
horrendous mysterious corpse frosting 43036
hairy iced enchantment sourstick 30752
bloodcurdling green genie liquorice 18465
shocking triple fudge enchantment butter 98337
unnerving iced cadaver pumpkin 81953
creepy possibly-poisonous devil candy bar 10272
creepy crushed witchcraft cake 30764
creepy iced phantasm taffy 53297
bloodcurdling magical vampire candy bar 4146
horrendous diabetic nighmarish halloween pure sucrose 8244
spine-chilling foul-looking wraith pastry 75837
creepy poisonous-looking corpse pie 94274
horrifying elemental fangs fudge 18500
spine-chilling crushed spirit cake 38981
alarming triple fudge grave donut 75846
frightening oozing zombie sugar 75852
unnerving orange-flavored spirit creme de la crumb 26703
horrifying orange-flavored warlock mix 63573
hair-raising iced witchcraft gummy 6233
hairy green skeleton donut 63586
eerie possibly-poisonous cobweb pastry 86117
creepy poisonous-looking enchantment frosting 73839
spooky mysterious skull fudge 59507
horrifying triple fudge skeleton pure sucrose 82040
unnerving elemental cobweb caramel apple 8314
horrendous oozing moonlight pastry 28795
hair-raising double chocolate halloween pure sugar 16508
hairy triple fudge witchcraft cottoncandy 57469
unnerving brain melting halloween mix 73859
eerie diabetic nighmarish hocus pocus candy bar 39054
unnerving elemental grim pumpkin 90258
spine-chilling possibly-poisonous witch donut 59542
horrifying diabetic nighmarish skeleton cottoncandy 28827
shocking diabetic nighmarish fangs doughnut 67744
hairy green enchantment cottoncandy 77988
creepy mysterious hocus pocus cottoncandy 90282
spooky oozing poltergeist chocolate 69803
eerie quadruple rasberry afterlife mix 28845
bloodcurdling crushed witch pumpkin 88249
unnerving double chocolate skeleton chocolate 47293
spine-chilling triple fudge moonlight cake 59583
bloodcurdling double chocolate death liquorice 88255
unnerving poisonous-looking cadaver bars 45251
hair-raising foul-looking zombie pure sugar 98504
horrifying magical phantom chocolate 14537
unnerving foul-looking fairy cake 61641
unnerving glucose exploding crypt pure sugar 90313
creepy magical spider frosting 39117
unnerving triple fudge spell pie 39123
unnerving mysterious fairy fudge 22743
creepy diabetic nighmarish moonlight candy bar 45274
spine-chilling glucose exploding corpse caramel apple 227
bloodcurdling orange-flavored cemetery sugar 82148
hairy green cobweb pastry 28910
unnerving fireball zombie frosting 47359
creepy iced ogre taffy 39168
shocking quadruple rasberry genie donut 37134
intimidating iced tombstone pastry 26895
frightening crushed jack-o-lantern liquorice 22801
eerie crushed enchantment caramel apple 76050
alarming elemental jack-o-lantern pie 43289
alarming poisonous-looking tomb pumpkin 33055
chilling iced devil doughnut 55586
horrifying double chocolate scarecrow taffy 96547
horrifying crushed pumpkin sugar 98598
bloodcurdling magical grim sourstick 78123
alarming oozing poltergeist pastry 96556
shocking fireball fangs pie 69934
eerie orange-flavored gravestone pure sucrose 67888
unnerving green afterlife taffy 67897
shocking quadruple rasberry bogeyman liquorice 314
horrifying foul-looking mausoleum caramel apple 51519
shocking crushed coffin treat 2376
unnerving diabetic nighmarish cemetery candy bar 331
hairy elemental ghost sourstick 96590
spine-chilling fireball witchcraft taffy 33104
unnerving crushed gravestone cake 78162
horrendous triple fudge mummy fudge 88408
intimidating magical fairy creme de la crumb 29017
hair-raising mysterious demon caramel apple 84323
horrifying iced cadaver cake 45416
hair-raising orange-flavored phantasm creme de la crumb 76137
creepy definitely-poisonous warlock frosting 65897
eerie double chocolate mausoleum creme de la crumb -1673
frightening magical werewolf taffy 67960
spine-chilling orange-flavored scarecrow pure sugar 76163
hairy triple fudge phantasm mix 70026
spooky elemental demon chocolate 406
spooky brain melting warlock mix 53658
shocking green witch pure sugar 51620
creepy double chocolate witch sugar 37289
eerie mysterious halloween treat 68010
spooky oozing grim bars -3659
spooky fireball fangs donut -3651
horrifying triple fudge tombstone pumpkin 96702
horrendous diabetic nighmarish mausoleum pumpkin 33218
intimidating glucose exploding death taffy 37322
spooky foul-looking zombie chocolate 12748
creepy quadruple rasberry grim mix 72143
spooky magical halloween cottoncandy 82384
hair-raising magical howl candy bar 45529
alarming brain melting witch bars 80346
horrendous diabetic nighmarish jack-o-lantern bars 53725
frightening iced mummy creme de la crumb 98783
creepy triple fudge halloween caramel apple 84448
hair-raising quadruple rasberry cemetery mix 96738
alarming quadruple rasberry scarecrow fudge 82406
frightening quadruple rasberry grim donut 84455
horrendous definitely-poisonous mist taffy 74225
spine-chilling possibly-poisonous ogre liquorice 78322
horrendous green phantom cottoncandy 37364
spine-chilling quadruple rasberry pumpkin pastry 51701
intimidating iced witch pie 6647
frightening foul-looking afterlife fudge 27128
horrendous orange-flavored halloween sucker 25081
chilling quadruple rasberry mist cake 51715
bloodcurdling elemental troll gummy 35339
creepy orange-flavored mummy mix 33298
spooky green gravestone sucker 4629
hair-raising brain melting phantasm pumpkin 31255
hair-raising mysterious mist pure sucrose 61976
horrifying elemental tombstone pie 27159
hair-raising magical ghoul treat 90647
chilling green cobweb pastry 39454
spine-chilling triple fudge werewolf caramel apple -1501
shocking double chocolate ogre liquorice 16937
alarming elemental moonlight donut 6702
hairy magical ghoul candy bar 68146
shocking foul-looking zombie fudge 37427
creepy glucose exploding werewolf sugar 14901
eerie green tarantula pure sugar -1478
unnerving double chocolate tomb doughnut 2619
spine-chilling poisonous-looking pumpkin pure sucrose 96833
intimidating vomit-inducing phantom bars 14913
spine-chilling quadruple rasberry death frosting 2633
hairy double chocolate warlock pie 23115
horrendous possibly-poisonous moonlight caramel apple 33362
horrendous poisonous-looking spell cake 21075
shocking mysterious spider pie 2650
hair-raising magical tombstone liquorice 6750
hairy magical fairy taffy 6753
hairy magical scarecrow cottoncandy 6757
hairy green spirit pumpkin -3482
bloodcurdling elemental goblin sucker 96883
eerie vomit-inducing poltergeist pure sucrose 29299
frightening double chocolate wraith sugar 74356
chilling magical tombstone pie 17018
frightening glucose exploding spell butter 23165
shocking iced grave mix 10882
frightening definitely-poisonous corpse pie 82568
alarming iced werewolf taffy 12936
bloodcurdling crushed phantasm frosting 19084
bloodcurdling glucose exploding fairy treat 60045
alarming possibly-poisonous witch fudge 60050
chilling green witch creme de la crumb -3437
hair-raising green jack-o-lantern mix 19098
bloodcurdling definitely-poisonous fairy mix 66202
horrendous orange-flavored warlock bars 74402
bloodcurdling double chocolate halloween taffy 80546
hairy magical zombie liquorice 76455
alarming foul-looking nightmare bars 45737
alarming glucose exploding afterlife cottoncandy 12973
intimidating green witch candy bar 98990
hairy double chocolate tomb sucker 43696
hair-raising fireball wraith sucker 45746
unnerving fireball afterlife doughnut 78529
unnerving quadruple rasberry mummy cottoncandy 47809
hair-raising foul-looking tombstone treat 4805
eerie poisonous-looking spider pastry 21189
hairy poisonous-looking coffin mix 23244
hair-raising crushed hocus pocus taffy 51917
horrendous magical tarantula chocolate 92884
creepy poisonous-looking grave sourstick 76505
spooky definitely-poisonous ogre doughnut 53977
spooky iced grim frosting 2780
unnerving iced cemetery candy bar 58079
unnerving triple fudge phantasm cottoncandy 66282
hairy double chocolate crypt pie 19179
spine-chilling orange-flavored cobweb caramel apple 86764
spine-chilling foul-looking scarecrow fudge 41713
spine-chilling crushed ghoul cake 82674
chilling fireball grave frosting 76531
frightening diabetic nighmarish afterlife treat 33528
bloodcurdling green goblin pie 94973
unnerving orange-flavored phantasm liquorice 19203
eerie brain melting mausoleum sucker 8964
spooky green wraith bars 84741
horrifying glucose exploding ghoul doughnut 45829
hair-raising glucose exploding grave liquorice 11015
creepy iced howl fudge 82697
hairy diabetic nighmarish mummy bars 88849
frightening iced potion liquorice 43796
bloodcurdling oozing warlock cake 19221
intimidating crushed cadaver caramel apple 84759
alarming mysterious ogre donut 80664
intimidating possibly-poisonous wraith liquorice 54051
chilling fireball witchcraft sugar 84771
shocking poisonous-looking halloween pastry 45859
hair-raising crushed vampire doughnut 29484
spine-chilling double chocolate ghoul bars 9010
bloodcurdling elemental corpse doughnut 19268
horrifying possibly-poisonous monster chocolate 56133
unnerving double chocolate devil cake 11078
frightening iced mausoleum treat 68423
spooky diabetic nighmarish tomb donut 4935
horrifying foul-looking spirit pie 2892
horrendous double chocolate nightmare sourstick 9037
creepy double chocolate ghost pure sugar 54106
horrendous definitely-poisonous tarantula pure sucrose 45917
bloodcurdling vomit-inducing cadaver gummy 52064
unnerving definitely-poisonous afterlife sucker 15211
alarming crushed poltergeist butter 29547
frightening iced grave taffy 95091
shocking green spider candy bar 35701
creepy triple fudge werewolf candy bar 60279
hair-raising mysterious tombstone sucker 80760
spooky double chocolate coffin fudge 39805
hairy oozing monster cake 50049
spine-chilling green fairy gummy 95114
spooky glucose exploding afterlife bars 11148
frightening poisonous-looking ghost pie 52109
eerie iced nightmare sucker 11154
shocking triple fudge scarecrow sugar 97175
alarming brain melting halloween donut 27543
spooky foul-looking witchcraft frosting 33692
frightening mysterious bogeyman caramel apple 31646
chilling oozing potion treat 60322
creepy fireball monster pure sucrose 76713
unnerving vomit-inducing demon caramel apple 29610
frightening iced grave gummy -1092
chilling possibly-poisonous skeleton cottoncandy 76737
hairy vomit-inducing vampire fudge 41922
chilling quadruple rasberry skull sucker 86982
bloodcurdling diabetic nighmarish jack-o-lantern liquorice 80840
hair-raising triple fudge cadaver butter -3122
frightening poisonous-looking genie treat -3121
spine-chilling green witch gummy 84952
eerie foul-looking spider pure sugar 99299
unnerving fireball fangs chocolate 62447
frightening oozing crypt mix 50166
frightening definitely-poisonous skull pure sugar 17401
horrendous oozing moonlight cottoncandy 87037
chilling oozing skeleton sugar 48126
eerie poisonous-looking enchantment chocolate 1021
alarming vomit-inducing witchcraft frosting 54270
unnerving elemental spider chocolate 78852
intimidating brain melting fairy mix 89095
alarming definitely-poisonous tomb pure sucrose 68619
bloodcurdling poisonous-looking tombstone gummy 99343
creepy mysterious witchcraft pure sucrose 87061
alarming mysterious tombstone pie 64537
horrendous magical goblin donut 33818
alarming vomit-inducing phantom sucker 89114
intimidating fireball ghost doughnut 95260
spooky iced moonlight treat 52252
spooky elemental zombie taffy 29726
unnerving crushed mausoleum donut 58408
creepy possibly-poisonous hocus pocus candy bar 54317
horrifying vomit-inducing halloween frosting 85050
shocking vomit-inducing demon liquorice 95291
intimidating foul-looking demon liquorice 91196
hair-raising diabetic nighmarish ghost sucker 52288
horrifying double chocolate bogeyman doughnut 42050
creepy orange-flavored bogeyman bars 70725
hairy possibly-poisonous skull cottoncandy 99398
bloodcurdling definitely-poisonous goblin candy bar 72778
creepy oozing tombstone sugar 97356
intimidating diabetic nighmarish poltergeist chocolate 48213
alarming brain melting cemetery frosting -934
spooky elemental tomb sourstick 37979
spooky glucose exploding demon pure sugar 1123
horrendous definitely-poisonous halloween butter 29797
horrendous poisonous-looking mausoleum pure sugar 64618
hairy orange-flavored werewolf butter 46187
intimidating brain melting afterlife pie 46193
chilling poisonous-looking warlock mix 76914
intimidating magical goblin taffy 15475
shocking green vampire doughnut 76932
bloodcurdling elemental ghost frosting -883
eerie poisonous-looking nightmare gummy 56469
eerie brain melting enchantment pumpkin 58517
creepy definitely-poisonous goblin chocolate 35996
frightening fireball wraith sugar 83101
bloodcurdling elemental spider chocolate 38048
frightening brain melting gravestone doughnut 25763
shocking elemental cobweb chocolate 31907
intimidating possibly-poisonous cobweb doughnut 97448
hairy mysterious grave butter -2900
shocking brain melting mummy pure sucrose -4945
intimidating mysterious ghost pure sucrose 44213
intimidating definitely-poisonous mummy butter -2889
hair-raising elemental corpse treat 87235
spine-chilling diabetic nighmarish halloween taffy 11468
hair-raising quadruple rasberry crypt chocolate 7384
alarming fireball ogre taffy 99544
horrendous quadruple rasberry cobweb pure sucrose 99550
spine-chilling vomit-inducing poltergeist sugar 52449
hairy orange-flavored moonlight taffy 23781
frightening orange-flavored cemetery fudge 56560
alarming triple fudge witch fudge 11506
frightening possibly-poisonous tomb pie 64754
spooky possibly-poisonous scarecrow sourstick 66804
creepy triple fudge spell chocolate 99583
horrendous glucose exploding crypt mix 77056
intimidating elemental scarecrow cottoncandy 42241
intimidating oozing grave pastry 9474
spine-chilling double chocolate spirit pie 3337
alarming crushed mummy fudge 95500
horrendous definitely-poisonous fangs pure sugar 19726
spine-chilling quadruple rasberry tombstone sugar 79118
intimidating crushed ogre bars 38160
spooky double chocolate genie bars 44305
unnerving vomit-inducing skeleton butter 29977
horrendous foul-looking vampire donut 83234
unnerving triple fudge vampire sugar 64808
hairy elemental poltergeist bars 32044
hair-raising double chocolate potion sugar 21805
spine-chilling possibly-poisonous tomb candy bar 44332
unnerving definitely-poisonous witch sourstick 9522
bloodcurdling vomit-inducing haunted cake 66870
spine-chilling mysterious scarecrow chocolate 5436
eerie iced cemetery mix 99645
hairy mysterious cadaver gummy 97597
alarming crushed grim cottoncandy 56646
bloodcurdling brain melting demon mix 93513
horrendous possibly-poisonous fairy pastry 21842
frightening poisonous-looking grave frosting 79189
intimidating quadruple rasberry witch fudge 11606
chilling poisonous-looking mummy taffy 62810
horrifying orange-flavored corpse doughnut 77158
alarming mysterious witch liquorice 99693
eerie triple fudge howl bars 85358
unnerving mysterious crypt liquorice 89461
bloodcurdling brain melting wraith pumpkin 71030
shocking double chocolate cemetery butter 48507
creepy oozing cemetery liquorice 71038
creepy double chocolate hocus pocus cottoncandy -4720
hairy brain melting bogeyman treat 3474
spine-chilling oozing tomb pure sucrose 58783
unnerving iced werewolf butter 36257
intimidating possibly-poisonous spider doughnut 34211
spine-chilling crushed zombie pure sugar -2653
chilling oozing corpse gummy 73128
unnerving orange-flavored tomb pure sucrose 46508
frightening orange-flavored corpse sucker 69040
creepy fireball moonlight pure sucrose 48561
alarming poisonous-looking bogeyman liquorice 73137
bloodcurdling poisonous-looking gravestone taffy 75187
hair-raising orange-flavored monster cottoncandy 95667
shocking poisonous-looking fangs pure sucrose 99773
spine-chilling orange-flavored vampire cake 30146
horrifying green mausoleum sucker 38341
eerie brain melting phantasm sucker 42448
bloodcurdling diabetic nighmarish grave treat 62928
hair-raising possibly-poisonous enchantment sugar 3539
hair-raising triple fudge mist creme de la crumb 73172
shocking quadruple rasberry hocus pocus gummy 11738
horrendous crushed jack-o-lantern chocolate 15834
horrendous definitely-poisonous skull pure sucrose 24028
chilling possibly-poisonous cobweb pie 26080
hair-raising crushed gravestone sourstick 67041
bloodcurdling definitely-poisonous coffin donut 77284
chilling iced haunted cake 69093
alarming definitely-poisonous crypt taffy 40424
eerie definitely-poisonous spirit donut 52713
shocking foul-looking cobweb sugar 79338
bloodcurdling fireball haunted treat 85483
hairy possibly-poisonous monster chocolate 26089
horrendous diabetic nighmarish vampire butter 21997
horrendous vomit-inducing devil caramel apple 21998
chilling glucose exploding scarecrow cottoncandy 93673
spine-chilling double chocolate witch doughnut 62965
eerie orange-flavored witch pure sucrose 67062
frightening foul-looking troll fudge -2571
horrendous vomit-inducing grave sugar 85499
unnerving oozing genie pure sucrose -2564
frightening triple fudge fangs caramel apple 65032
horrifying quadruple rasberry crypt donut 50699
hair-raising poisonous-looking mausoleum frosting 44556
eerie possibly-poisonous crypt fudge 34317
horrifying glucose exploding gravestone fudge 91664
horrifying diabetic nighmarish fangs candy bar 83475
spine-chilling quadruple rasberry monster doughnut 67103
chilling orange-flavored howl chocolate 48673
creepy possibly-poisonous haunted cake 38438
hair-raising vomit-inducing tomb pastry 89655
frightening double chocolate howl treat 97854
spine-chilling magical monster cottoncandy 40512
chilling foul-looking phantom mix 65097
frightening definitely-poisonous afterlife liquorice 46674
alarming iced fangs sourstick 44626
frightening definitely-poisonous corpse treat 58965
creepy quadruple rasberry warlock frosting 44632
horrendous foul-looking crypt sourstick 18012
spooky vomit-inducing ogre caramel apple 3680
shocking definitely-poisonous grave fudge -2452
alarming brain melting halloween pie 52853
bloodcurdling possibly-poisonous phantasm cake 7799
frightening crushed ghoul pure sucrose 28282
bloodcurdling vomit-inducing devil cake 1659
hairy double chocolate werewolf sugar 85634
bloodcurdling diabetic nighmarish spirit doughnut 52866
hair-raising foul-looking jack-o-lantern mix 71303
spine-chilling iced coffin bars 71304
horrendous triple fudge tombstone frosting 16008
horrifying crushed ghost sugar -4473
eerie oozing spider donut 61067
shocking foul-looking gravestone pumpkin 93838
eerie iced coffin taffy 18062
bloodcurdling glucose exploding tomb cake 95892
unnerving quadruple rasberry phantasm cottoncandy 59028
spine-chilling glucose exploding pumpkin pure sucrose -2398
intimidating oozing afterlife liquorice 77476
shocking foul-looking zombie pie 65189
spooky glucose exploding jack-o-lantern pure sugar 42662
frightening poisonous-looking moonlight chocolate 48807
hair-raising glucose exploding jack-o-lantern cottoncandy 3765
bloodcurdling mysterious afterlife sucker 54969
unnerving mysterious goblin pure sugar 9914
hair-raising poisonous-looking skeleton creme de la crumb 54973
unnerving mysterious goblin sucker 18112
horrendous fireball scarecrow cake 73411
spooky mysterious tarantula taffy 20169
creepy crushed death cake 42706
creepy quadruple rasberry cadaver sourstick 81622
horrendous oozing devil fudge 26327
eerie glucose exploding troll sugar 52950
alarming triple fudge demon sourstick 30425
bloodcurdling fireball afterlife sugar 55002
shocking vomit-inducing potion caramel apple 5847
spine-chilling iced afterlife pie 67292
creepy iced halloween pumpkin 63191
bloodcurdling double chocolate fangs butter 83681
horrendous possibly-poisonous devil cottoncandy 89826
intimidating crushed spell doughnut -286
horrendous foul-looking grim sourstick 73446
unnerving foul-looking witchcraft pastry 9959
shocking quadruple rasberry grim mix 75510
chilling triple fudge monster pie 24315
creepy magical mummy pie 67330
horrifying elemental tarantula treat 89860
shocking oozing afterlife mix -4340
horrifying vomit-inducing goblin sucker 71437
shocking triple fudge phantom pumpkin 36622
eerie orange-flavored fangs candy bar 96016
alarming crushed phantom candy bar 69393
bloodcurdling double chocolate spirit cottoncandy 14103
horrendous mysterious witch creme de la crumb 32539
hair-raising possibly-poisonous tarantula doughnut -2271
horrifying double chocolate witch fudge 28453
creepy poisonous-looking coffin pumpkin 50982
spooky definitely-poisonous bogeyman fudge 57127
horrifying poisonous-looking warlock sourstick 5936
spooky glucose exploding pumpkin sucker 34624
spooky iced cemetery bars 81732
spooky definitely-poisonous halloween sourstick 57162
bloodcurdling triple fudge afterlife pie 71502
eerie elemental bogeyman sucker 12112
eerie green ghost sucker 87888
alarming foul-looking crypt liquorice -4270
frightening foul-looking mausoleum pure sugar 75615
spooky fireball witch pure sugar -160
unnerving iced vampire taffy 96114
unnerving definitely-poisonous tarantula sourstick 94072
horrendous oozing coffin donut 53113
eerie foul-looking devil pure sugar 67450
bloodcurdling orange-flavored cemetery doughnut 67458
unnerving definitely-poisonous spider bars 34693
shocking definitely-poisonous cadaver creme de la crumb 40839
intimidating orange-flavored grave donut 26507
hair-raising quadruple rasberry ghost cottoncandy 59278
creepy crushed pumpkin sucker 71567
hair-raising orange-flavored potion caramel apple 20367
hairy glucose exploding spider doughnut 40855
unnerving oozing wraith caramel apple 55196
horrifying poisonous-looking mausoleum frosting 18335
intimidating fireball poltergeist sugar 63409
spine-chilling vomit-inducing enchantment pure sugar 34741
shocking elemental genie liquorice 32694
intimidating vomit-inducing fangs sucker 83898
creepy iced cemetery cake 81854
unnerving crushed moonlight pie 61384
shocking definitely-poisonous hocus pocus cottoncandy 94156
horrendous quadruple rasberry devil mix 55246
intimidating iced demon mix 2002
eerie poisonous-looking cadaver pure sucrose 53204
creepy triple fudge skull creme de la crumb 55257
unnerving quadruple rasberry poltergeist butter -4134
hair-raising double chocolate potion sourstick 26602
alarming possibly-poisonous hocus pocus chocolate 18421
shocking orange-flavored demon liquorice 26613
frightening magical afterlife butter 92152
horrendous poisonous-looking troll pastry 12281
