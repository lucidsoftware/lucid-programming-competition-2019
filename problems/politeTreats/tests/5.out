horrifying double chocolate potion sucker
